embedchain/config/InitConfig.py

```diff
@@ -27,6 +29,11 @@ def __init__(self, ef=None, db=None):
             self.db = ChromaDB(ef=self.ef)
         else:
             self.db = db
+            
+        if id is None:
+            self.id = str(uuid.uuid4())
+        else:
+            self.id = id
```

Comment:
Can rewrite the whole thing in one line:

```python
self.id = id if id is not None else str(uuid.uuid4())
```

Reply:
Removed uuid altogether